tests/integration/condition__collection__have_texts_test.py

```diff
@@ -24,17 +24,109 @@
 
 import os
 
+import pytest
+
 from selene import have
+from selene.core.exceptions import TimeoutException
+from selene.support.shared import browser
 from tests.integration.helpers.givenpage import GivenPage
 
 
 def test_unicode_text_with_array(session_browser):
     GivenPage(session_browser.driver).opened_with_body(
         '''
-        <ul>Привет:
-           <li>Саше</li>
-           <li>Яше</li>
+        <ul>Hello:
```

Comment:
todo: In case we change unicode to common english examples, we should rename the test `test_unicode_text_with_array` to something more meaningful like other tests in module.

thought: should we create another unicode test in this case?

Reply:
maybe yes, we can duplicate these tests for Unicode, idk if it's better to create them in tests/integration/condition__collection__have_texts_unicode_test.py or no?